test/units/modules/remote_management/oneview/test_oneview_storage_system.py

```diff
@@ -0,0 +1,300 @@
+# Copyright (c) 2016-2017 Hewlett Packard Enterprise Development LP
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+import pytest
+import yaml
+
+from ansible.compat.tests import mock
+from hpe_test_utils import mock_ov_client, mock_ansible_module
```

Comment:
It is considered a bad practice to import from pytest, I'm also seeing several flake errors commenting that these are being rewritten.